zynqmp.mk

```diff
@@ -1,8 +1,8 @@
 ROOT			= $(PWD)/..
 PLATFORM		?= zcu102
-PETALINUX_PATH		?= /opt/Xilinx/petalinx_2018_2
+PETALINUX_PATH		?= $(PETALINUX)
 BSP_PATH		?= ./xilinx-${PLATFORM}-v2018.2-final.bsp
```

Comment:
Will this still have a "static" value (`v2018.2-final`)? Shouldn't that also leverage `$(PETALINUX_VER)`?

Reply:
updated